src/defaultConfig.js

```diff
@@ -4,6 +4,52 @@ const slugFilter = require("./Filters/Slug");
 const slugifyFilter = require("./Filters/Slugify");
 const getCollectionItem = require("./Filters/GetCollectionItem");
 
+/**
+ * @module 11ty/eleventy/defaultConfig
+ */
+
+/**
+ * @callback addFilter - Register a global filter.
+ * @param {string} name - Register a template filter by this name.
+ * @param {function} callback - The filter logic.
+ */
+
+/**
+ * @typedef {Object} config
+ * @property {addFilter} addFilter - Register a new global filter.
+ */
+
+/**
+ * @typedef {Object} defaultConfig
+ * @property {Array<String>} templateFormats - An array of accepted template formats.
+ * @property {String} [pathPrefix='/'] - The directory under which all output files should be written to.
```

Comment:
Q: Why lowercase `{string}` above on L13 and uppercase `{String}` here on L25 (and below)?

Reply:
Should be lowercase: https://jsdoc.app/tags-type.html
Commit incoming …